codemods/videos.js

```diff
@@ -1,22 +1,36 @@
 const visit = require('unist-util-visit');
 const {
   addAttribute,
-  hasClassName,
   findAttribute,
+  isMdxBlockElement,
 } = require('./utils/mdxast');
 
+const getVideoProps = (src) => {
+  const propsObj = {};
+  const domain = src.split('/')[2];
```

Comment:
[nit] There is also a [`URL`](https://nodejs.org/docs/latest-v12.x/api/url.html) module you can create to get url information. If you like not having to regex or split on the URLs yourself, that module can come in real handy since its pretty easy to use and extract URL information out of.

```js
const url = new URL(src)

if (url.hostname === 'fast.wistia.net') {
  // ...
}
```

Reply:
yes! i was contemplating using a module, but wasn't sure if that would preferable to just splitting the string. i'll update to use that